en-us/update-instructions.md

```diff
@@ -86,12 +86,9 @@ Please __do not upgrade from stable to alpha/beta__, use a separate instance. An
 
 ### Prism Launcher
 
-1. Click `Add Instance`.
-2. Select `Modrinth` tab from the left
-3. Select "Fabulously Optimized"
-4. Click `OK`
-5. Once the update prompt comes up, click `Update existing instance`
-6. Once the name prompt comes up, click `Yes`
+1. Click `Edit` in the sidebar
+2. Select `Modrinth` on the tab list
+3. Click `Update pack`
```

Comment:
No need to select it?

Reply:
I actually meant to ask whether it is necessary to select the latest version before clicking "update pack".